modules/doobie/src/test/scala/tofu/doobie/example/TofuDoobieExample.scala

```diff
@@ -34,9 +33,11 @@ final case class Ctx(traceId: String)
 trait Logging[F[_]] {
   def info(msg: String): F[Unit]
 }
-object Logging      {
-  def make[F[_]: FlatMap: Console: HasContext[*[_], Ctx]]: Logging[F] =
+
+object Logging {
```

Comment:
As was told in tofu-chat (or ponv, I don't remember exactly where), this is example feels overloaded and hides the demonstrated techniques.
Is it possible to split it up a bit, like move Logging and Tracing into separate file `infrastructure.scala`? That would make it easier to read and such.